docusaurus/docs/extensions/api/nav/custom-page.md

```diff
@@ -9,7 +9,7 @@ As we've seen from the previous chapter, a developer can register a top-level pr
 const BLANK_CLUSTER = '_';
 
 
-export function init($plugin, store) {
+export function init($plugin: IPlugin, store: any) {
```

Comment:
I believe we need to `import { IPlugin } from '@shell/core/types'`. We should add it to all places, imho.

Reply:
Good catch. I fixed it.